cernopendata/api.py

```diff
@@ -174,3 +201,38 @@ def dumps(self):
     def flush(self):
         """Flushing the information into the object."""
         return self.dumps()
+
+
+class FileObjectCold(FileObject):
+    """Overwrite the fileobject to get multiple URI and availability."""
```

Comment:
explain a bit better
```suggestion
    """Overwrite the fileobject to allow multiple URI for the same instance. For example, a file could be located both on eos disk and on tape. The class introduces as well the availability concept: if a file is available only on tape, then the file has to be requested before users can access it."""
```